wom/src/main/scala/wom/values/WdlObject.scala

```diff
@@ -1,13 +1,14 @@
-package wdl.values
+package wom.values
 
-import wdl.types._
-import wdl.util.FileUtil
-import wdl.{TsvSerializable, WdlCall}
+import wom.TsvSerializable
+import wom.types._
+import wom.util.FileUtil
 
 import scala.util.{Failure, Success, Try}
 
 trait WdlObjectLike {
   def value: Map[String, WdlValue]
+  def copyWith(values: Map[String, WdlValue]): WdlValue with WdlObjectLike
```

Comment:
part of the sneakiness required to copy `WdlCallOutputsObject`s from WOM code

Reply:
I might ask you to tech talk what this sneakiness is all about

Reply:
This is for `WdlValue.takeMaxValues`